### PR TITLE
[TA] Live test with AAD

### PR DESCRIPTION
**Pull request by mssfang**
Live Test enabled with AAD.

### CODE REVIEW

File: sdk/textanalytics/tests.yml [hunk 1]
```diff
@@ -4,13 +4,8 @@ stages:
...
+      TestMode: RECORD # to make sure recordedData is not null but still testing LIVE mode
       Artifacts:
         - name: azure-ai-textanalytics
           groupId: com.azure
           safeName: azureaitextanalytics
-      # resource creation is disabled because we are trying to test the stable endpoint.
-      # Will enable it when service rolls out AAD to all region.
-      DisableAzureResourceCreation: true
-      EnvVars:
-        AZURE_TEXT_ANALYTICS_API_KEY: $(java-textanalytics-test-api-key)
```
Comment (deyaaeldeen) [inline sdk/textanalytics/tests.yml:15]: why deleting the env vars?

Comment (mssfang) [reply sdk/textanalytics/tests.yml:15]: it for stable endpoint, which can't not delete and then created.

**Review by kristapratico — APPROVED** [review]

**Review by samvaity — APPROVED** [review]

**Pull request merged**